lib/ansible/module_utils/common/errors.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2018, Ansible Project
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class AnsibleTimeoutError(Exception):
+    pass
```

Comment:
I don't think we want common/errors because the project management of module_utils doesn't lend itself to central planning.  As I said in slack we need to move timeout itself into common.

Reply:
a new timeout maybe, the current one is fact gathering specific

Reply:
It's not fact gathering specific.  I would improve it when moving it over, though.